starknet_py/net/gateway_schemas/gateway_schemas.py

```diff
@@ -206,10 +245,15 @@ def make_dataclass(self, data, **kwargs):
                 )
                 storage_diffs.append(storage_diff)
 
+        declared_contracts = data["state_diff"]["declared_contracts"]
```

Comment:
I'd prefer this had its' own schema like `ContractDiffSchema` and `StorageDiffSchema` even if it's just a single field.